posthog/queries/app_metrics/app_metrics.py

```diff
@@ -121,25 +128,27 @@ def run(self):
 class AppMetricsErrorDetailsQuery:
     QUERY = QUERY_APP_METRICS_ERROR_DETAILS
 
-    def __init__(self, team: Team, plugin_config_id: int, filter: AppMetricsErrorsRequestSerializer):
+    def __init__(self, team: Team, plugin_config_id: int, category: str, error_type: str, job_id: str | None = None):
         self.team = team
         self.plugin_config_id = plugin_config_id
-        self.filter = filter
+        self.job_id = job_id
+        self.category = category
+        self.error_type = error_type
```

Comment:
Nit: With initialization like this, the class would be a bit nicer developer experience-wise if it used dataclasses (`@dataclass` decorator)